index.js

```diff
@@ -24,6 +26,26 @@ sysLogLevels = {
   },
 };
 
+/**
+ * setups up formatters for winston 3 format (https://github.com/winstonjs/winston#formats)
```

Comment:
This function sets up the formatters in the winston 3 format of using functions. I did not setup a stringify option because it worked the same as JSON, however; since some repos use stringify = null to prettyprint, I added that logic below.